facebook/react-native
# fix:TextInput textBreakStrategy default value on Android (Paper - old arch)

## Summary:



This pull request addresses an issue related to the TextInput component. Currently, the default value for the textBreakStrategy prop is set to highQuality on Android platform. This is against the expectation as per RN documentation [here](https://reactnative.dev/docs/textinput#textbreakstrategy-android). 

This PR proposes a fix that updates the default value of textBreakStrategy to simple on Paper. For new arch the value seems to be inherited from ParagraphAttributes and is set to `TextBreakStrategy::HighQuality` by default. Overriding the value and setting it to `Simple` for TextInput seems to be beyond my capabilities. 

fixes https://github.com/facebook/react-native/issues/38244

## Changelog:



[ANDROID] [FIXED] - Set default value of textBreakStrategy to 'simple' on Paper (old-arch)


## Test Plan:



- Run the RNTester App
- Navigate to examples for TextInput
- Ensure `breakStrategy` attribute for EditText components are 'simple' by default. Use LayoutInspector from Android Studio for the same.
